client/jms/pom.xml

```diff
@@ -130,6 +130,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>jcl-over-slf4j</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
```

Comment:
I only added this as it seemed like the dependency should be explicit rather than transit from `org.slf4j:jcl-over-slf4j`.